### DISCUSSION TITLE
Welcome to magic-mouse-js Discussions!

### DISCUSSION THREAD

**Discussion by dshongphuc** [dshongphuc/magic-mouse-js#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jasonhsantiago** [comment]
Is there an option for changing the color of the cursor?

**Reply by dshongphuc** [reply]
Hi @jasonhsantiago , no we don't have any option to change the color, you can simply change the color using CSS

**Reply by dshongphuc** [reply]
Check this file for class name reference: https://github.com/dshongphuc/magic-mouse-js/blob/master/src/scss/style.scss

**Comment by jasonhsantiago** [comment]
What’s the class name to change the color?Sent from my iPhoneOn May 13, 2024, at 2:20 AM, Phuc H. ***@***.***> wrote:﻿
Hi @jasonhsantiago , no we don't have any option to change the color, you can simply change the color using CSS

**Comment by jasonhsantiago** [comment]
Okay is there a way to make it so the system or original mouse disappears when hovering over links?I just want the magic mouse ti be showing - not other mouse. Sent from my iPhoneOn May 13, 2024, at 2:22 AM, Phuc H. ***@***.***> wrote:﻿
Check this file for class name reference: https://github.com/dshongphuc/magic-mouse-js/blob/master/src/scss/style.scss
